test/harness/testharness.js

```diff
@@ -132,7 +132,7 @@ policies and contribution forms [3].
     };
 
     WindowTestEnvironment.prototype._forEach_windows = function(callback) {
-        // Iterate of the the windows [self ... top, opener]. The callback is passed
+        // Iterate of the windows [self ... top, opener]. The callback is passed
```

Comment:
Nit: This strikes me as odd English. Shouldn't it be either "Iteration of the windows" or "Iterate the windows"?

Reply:
Yeah, I thought so as well, but I didn't want to get distracted further :)